packages/terra-core-docs/src/terra-dev-site/test/hyperlink/Test.module.scss

```diff
@@ -0,0 +1,8 @@
+:local {
+    .truncated {
+        width: 150px;
+        height: 50px;
+        border-style: dashed;
+        margin-bottom: 10px;
+    }
+  }
```

Comment:
If this is only used for a single file, can we use inline styles for this? e.g. 
https://css-tricks.com/different-ways-to-write-css-in-react/#aa-write-inline-styles

Reply:
Done.